applications/PoromechanicsApplication/tests/element_tests/undrained_soil_column_2D/ProjectParameters.json

```
@@ -47,7 +47,7 @@
         "nonlocal_damage":                    false,
         "characteristic_length":              0.05,
         "linear_solver_settings":             {
-            "solver_type":   "SuperLUSolver"
+            "solver_type":   "super_lu"
```

```suggestion
            "solver_type":   "ExternalSolversApplication.super_lu"
```
@ipouplana you are right, the application in which the linear-solver is defined should be prepended to the name of the solver (currently this is necessary for solvers from the ExternalSolversApplication and the EigenSolversApplication)

The reason for this is that then the python-linear-solver-factory can import the application that is needed and you don't have to import those apps in the MainScript
Pretty much the same as how the ProcessFactory works

---

Ok, thanks. I will adapt this PR accordingly